### ISSUE TITLE
Can't copy data when using Azurite

### ISSUE THREAD

**Issue by LoofahBu** [Azure/azure-storage-azcopy#867]
### Which version of the AzCopy was used? 
10.3.3

### Which platform are you using? (ex: Windows, Mac, Linux)
Linux

### What command did you run?
`azcopy copy http://azurite:10000/devstoreaccount1/sample/Cont_20190213/0000574133.jpg`
`azcopy copy http://localhost:10000/devstoreaccount1/sample/Cont_20190213/0000574133.jpg`

### What problem was encountered?
We use [Azurite](https://github.com/Azure/Azurite) as our local storage. A sidecar container wants to download data by invoking **azcopy**. But it always show `failed to parse user input due to error: the inferred source/destination combination is currently not supported. Please post an issue on Github if support for this scenario is desired`.

### How can we reproduce the problem in the simplest way?
1. Run **azurite** by docker
2. Prepare some fake data and generate data url through [AzureStorageExplorer](https://github.com/microsoft/AzureStorageExplorer)
3. Run a **ubuntu:16.04** container which should on the same docker network with **azurite** container
4. Install **azcopy**
5. Execute command

### Have you found a mitigation/solution?
No

**Comment by JohnRusk** [comment]
This looks like an issue that we are fixing here: https://github.com/Azure/azure-storage-azcopy/pull/743

The fix should be released in version 10.4.  (And to answer the obvious question, sorry, we don't pre-announce release dates).

**Comment by adreed-msft** [comment]
Hi there! This should be fixed in 10.4. Enjoy!

**Comment by JohnRusk** [comment]
BTW You will need to use the from-to parameter when using Azurite. AzCopy will tell you that when you run it, if you forget.

Here are some examples of how to use it:

```
azcopy copy /data/foo http://azurite:10000/devstoreaccount1/sample/foo --from-to LocalBlob
azcopy copy http://azurite:10000/devstoreaccount1/sample/foo /data/foo --from-to BlobLocal
```
